test/lib/Launcher.ts

```diff
@@ -105,10 +127,20 @@ export class Launcher {
 
     public stop(): void {
         this._processes.forEach((p) => {
-            p.connector.on("exit", () => {
-                p.webhookServer?.close();
-            });
-            p.connector.kill();
```

Comment:
this code made no sense. All you want to do here is

1. close the webhookServer
2. kill the connector

why not just 
```ts
p.webhookServer?.close();
p.connector.kill();
```

IMO your stopClient method is also overkill.

Reply:
The problem is that between killing the connector and closing the webhook server, there could be some event send. And if the webhook server is closed before, this would create an error in the connector

Reply:
But that's not problem, we don't care about any issues here. Additionally you can just call the methods the other way round.